examples/wschat.py

```diff
@@ -0,0 +1,37 @@
+from vyked import Host, WSService, ws, WebSocketResponse
+from aiohttp.web import MsgType
+
+
+class ChatWSService(WSService):
+
+    def __init__(self, ip, port):
+        super(ChatWSService, self).__init__("ChatService", "1", ip, port)
+        self.msgs = []
+
+    @ws(path='/')
+    def chat(self, request):
+        wsk = WebSocketResponse()
+        wsk.start(request)
+        while True:
```

Comment:
Is this design okay:

@ws(path='/')
def chat(msg):
   process_msg
   return response

This should hide the WebSocketResponse and the while loop from the service.